python/ray/data/datasource/file_meta_provider.py

```diff
@@ -313,22 +313,22 @@ def prefetch_file_metadata(
     ) -> Optional[List["pyarrow.parquet.FileMetaData"]]:
         from ray.data.datasource.file_based_datasource import _fetch_metadata_parallel
         from ray.data.datasource.parquet_datasource import (
+            FRAGMENTS_PER_META_FETCH,
             PARALLELIZE_META_FETCH_THRESHOLD,
-            PIECES_PER_META_FETCH,
             _fetch_metadata,
             _fetch_metadata_serialization_wrapper,
-            _SerializedPiece,
+            _SerializedFragment,
         )
 
         if len(pieces) > PARALLELIZE_META_FETCH_THRESHOLD:
             # Wrap Parquet fragments in serialization workaround.
-            pieces = [_SerializedPiece(piece) for piece in pieces]
+            pieces = [_SerializedFragment(piece) for piece in pieces]
```

Comment:
can you also change the variable `pieces`, `pieces` -> `fragment`, `fragments`? and also all other places? Thanks.

Reply:
Ah, good catch. Looks like I missed a few.

Fixed.